crypto/x509/x509_lu.c

```diff
@@ -333,42 +338,19 @@ int X509_STORE_add_cert(X509_STORE *ctx, X509 *x)
     if (added == 0)             /* obj not pushed */
         X509_OBJECT_free(obj);
     if (!ret)                   /* on push failure */
-        X509err(X509_F_X509_STORE_ADD_CERT, ERR_R_MALLOC_FAILURE);
+        X509err(func, ERR_R_MALLOC_FAILURE);
```

Comment:
Ah, this is a bad idea, `make update` will complain. Better to return specific values and have the calling functions add the error codes.